aws/resource_aws_neptune_parameter_group.go

```diff
@@ -288,4 +291,14 @@ func resourceAwsNeptuneParameterGroupDelete(d *schema.ResourceData, meta interfa
 		}
 		return nil
 	})
+
+	if isResourceTimeoutError(err) {
+		_, err = conn.DeleteDBParameterGroup(&deleteOpts)
+	}
+
+	if err != nil {
```

Comment:
We'll also want to ignore `ErrCodeDBParameterGroupNotFoundFault` here too:

```suggestion
	if isAWSErr(err, neptune.ErrCodeDBParameterGroupNotFoundFault, "") {
		return nil
	}

	if err != nil {
```